arch/arm/src/cxd56xx/cxd56_cpu1signal.c

```diff
@@ -188,8 +186,6 @@ int cxd56_cpu1siginit(uint8_t sigtype, void *data)
 
   priv->ndev++;
 
```

Comment:
Do we have any protection now for `priv->ndev` access done from multiple threads?

Reply:
According to the current implementation, "cxd56_cpu1siguninit" will only be called once during the NuttX startup phase, and it won't involve scenarios of multi-threaded concurrent access.